pkg/drivers/hyperkit/iso.go

```diff
@@ -45,36 +44,15 @@ func ExtractFile(isoPath, srcPath, destPath string) error {
 	}
 
 	dst, err := os.Create(destPath)
-	defer dst.Close()
 	if err != nil {
 		return err
 	}
+	defer dst.Close()
 
 	_, err = io.Copy(dst, f.Sys().(io.Reader))
 	return err
 }
 
-func readFile(isoPath, srcPath string) (string, error) {
```

Comment:
Is this just no longer needed?

Reply:
Correct. The linter identified this as an unused function, and it seems legit.